.coveragerc

```diff
@@ -1,2 +1,7 @@
 [run]
-omit = setup.py
+source =
+   metakb/
+   tests/
+omit =
+   setup.py
+   docs/*
```

Comment:
Suggest adding the following to ignore venv/ from coverage report
```     venv/*

```

Reply:
@bwalsh It looks like your suggestion isn't showing.

Reply:
Created a separate PR to add single line:

```   venv/*
```